src/components/wallet/modal/modal-body.tsx

```diff
@@ -0,0 +1,90 @@
+import { ReactElement } from 'react';
+import * as DialogPrimitive from '@radix-ui/react-dialog';
+import classNames from 'classnames';
+import { getModalMessage, wallets } from './wallets';
+import css from './modal.module.scss';
+import Icon from './icon';
+import CloseIconLight from '@assets/svg/modal/ic-close-light.svg';
+import CloseIconDark from '@assets/svg/modal/ic-close-dark.svg';
+
+export interface DialogContentProps {
+    children: ReactElement;
+}
+
+export const DialogContent = ({ children, ...props }: DialogContentProps) => (
```

Comment:
```suggestion
export const DialogContent = ({ children, ...props }: DialogContentProps) => (
```
Do we need to export this out since it is only used here? Will there be any case where the `DialogContent` will be used as an independent component?